README.md

```diff
@@ -167,3 +167,87 @@ At a minimum, Gazebo `v2.x` is required (which is installed by default with ROS
 
 1. `iris.sdf` can only be generated with Gazebo >= `v3.0`, as it requires use of the `gz sdf ...` tool. If this requirement is not met, you will not be able to use the Iris MAV in any of the simulations.
 2. The Gazebo plugins `GazeboGeotaggedImagesPlugin`, `LidarPlugin` and the `LiftDragPlugin` all require Gazebo >= `v5.0`, and will not be built if this requirement is not met.
+
+Extended Wind Plugin
+-------------------------
+
+The existing wind plugin in RotorS has been extended to allow the use of a custom, static wind field for a given world. This is done by enabling the wind plugin macro in the aircraft base file (in our case `techpod_base.xacro`, which is found in `rotors_simulator/rotors_description/urdf`), and feeding it the correct parameters and files. The wind grid and values are generated using a suitable model, and saved in a text file read by the plugin.
+
+#### Grid specifications
+
+The grid used to define the wind field must be equidistant in x, respectively y-direction. The points in z-direction are distributed with upwards linearly increasing distance (spacing factors can be tuned as desired). The grid is terrain-following, meaning that all the points in the lower z-layer have a constant altitude offset from the terrain.
+
+#### Wind field text file format
```

Comment:
Ok, done. I also added a short paragraph in the README documentation that explains how to use the Python script.